docs/Update-SSHKeys.md

```diff
@@ -58,21 +58,25 @@ Now with your new MachineConfig yaml file (using the example above):
     oc create -f example-ssh-update.yaml
 ```
 
-You should see the new MachineConfig name appear immediately, from our example config:
+You should see the new MachineConfig name appear almost immediately, from our example config:
 ```sh
     oc get machineconfigs
 
-    NAME                                      AGE
-    00-master                                 1h
-    00-worker                                 1h
-    01-master-kubelet                         1h
-    01-worker-kubelet                         1h
-    master-5f2f745c8182f910df286ccf01b9aecb   1h
-    master-79f1c73c1d9df168421612112efc1dde   1h
-    ssh-workers-01                            5s
-    worker-2d688eed7f057feb95d9beef6af93732   1h
-    worker-52df682dc5cb3976b063ef3f197ead5e   5s
-
+    NAME                                                        GENERATEDBYCONTROLLER             IGNITIONVERSION   CREATED
+    00-master                                                   4.0.0-alpha.0-9-gfc1a99e6-dirty   2.2.0             87m
+    00-master-ssh                                               4.0.0-alpha.0-9-gfc1a99e6-dirty                     87m
```

Comment:
00-master-ssh & 00-worker-ssh now have ignition versions 2.2.0, so we can put values in this column